lib/seahorn/BmcPass.cc

```diff
@@ -360,9 +361,9 @@ class BmcPass : public llvm::ModulePass {
           if (CexFileRef.endswith(".ll") || CexFileRef.endswith(".bc")) {
             auto &tli = getAnalysis<TargetLibraryInfoWrapperPass>();
             auto const &dl = F.getParent()->getDataLayout();
-            BmcTraceWrapper<ZBmcTraceTy> trace_wrapper(trace);
-            dumpLLVMCex(trace_wrapper, CexFileRef, dl, tli.getTLI(F),
-                        F.getContext());
+            cexGen::CexExeGenerator<ZBmcTraceTy> cex(trace, dl, tli.getTLI(F),
```

Comment:
I think at this stage it is better to keep both old and new functionality and have a command line switch to decide which one to use.

For the new functionality, we do not want to port all of the existing one -- only what makes sense in the current context. However, there are users of old functionality so we can't just remove it.